xbrowse_server/templates/family/family_home.html

```diff
@@ -145,12 +146,83 @@ <h4 class="section-header">Individuals</h4>
                         <td>{% if indiv.maternal_id %}{{ indiv.maternal_id }}{% else %}-{% endif %}</td>
                         <td>{% if indiv.paternal_id %}{{ indiv.paternal_id }}{% else %}-{% endif %}</td>
                         <td>{% for ipheno in indiv.get_phenotypes %}{{ ipheno.phenotype.name }}{% empty %}-{% endfor %}</td>
+                        <td><button id={{ indiv }}_{{ project.project_id }} type="button" class="btn btn-info btn-sm phenotipsEditModalBtn">Edit in Phenotips</button></td>
                     </tr>
+                    
+                    
                     {% endfor %}
                     </tbody>
                 </table>
             </div>
             
+
+
+        <!-- Phenotips integration -->
+		<div id="phenotipsModal" class="modal" role="dialog">
+			<div class="modal-dialog">
+				<div class="modal-content">
+					<div class="modal-header">
+						<button type="button" class="close" data-dismiss="modal">&times;</button>
+						<h4 class="modal-title">Phenotips</h4>
+					</div>
+					<div class="modal-body"">
+						<iframe name="phenotipsEditPatientFrame" id="phenotipsEditPatientFrame" src="" width=1000 height=800 frameborder="0"></iframe>
+					</div>
+				</div>
+			</div>
+		</div>
+		<script>
+		$('.phenotipsEditModalBtn').on('click',function(event) {
+			sessionStorage.setItem('currentInFocusIndividualId',this.id)
+			var patientDetails=this.id.split('_');
+			var patientId=patientDetails[0];
+			var project=patientDetails[1];
+			showEditPageForThisPatient(patientId,project);
+		});
+		</script>
+		
+		<!-- 
+		/**
+		* Temporarily inserting this contents into the HTML
+		*/
+		<script type='text/javascript' src='{{ STATIC_URL }}phenotips_integration/phenotips_integration.js'></script>
+    	<link rel='stylesheet' type='text/css' href='{{ STATIC_URL }}phenotips_integration/phenotips_integration.css' />    
+		 -->
+		<script>
+		/**
+		 * Given a patient ID (NAxxx,) show the phenotips edit page for it
+		 **/
+		function showEditPageForThisPatient(patientId,project) {
+			var uri = '/api/phenotips/proxy/edit/' + patientId + '?' + 'project=' + project;
+			$('#phenotipsEditPatientFrame').attr('src', uri);
+			$('#phenotipsModal').modal('show');
+		}
+		</script>
+		<style>
```

Comment:
yeah, I prefer it separate as well and that's how the code originally was as you found. But there is something different in the way xBrowse is serving static code that broke it. For testing I had re-inserted it.I will pull it out again soon. On the same note, at some point we need to stop xBrowse serving static content and outsource that piece to apache HTTPD www. For when we start scaling up.